client-ts/FunctionalTests/package.json

```diff
@@ -6,25 +6,23 @@
   "main": "index.js",
   "dependencies": {},
   "devDependencies": {
-    "@types/selenium-webdriver": "^3.0.8",
-    "@types/yargs": "^11.0.0",
+    "@types/node": "^9.4.6",
     "es6-promise": "^4.2.2",
-    "faucet": "0.0.1",
-    "selenium-webdriver": "^4.0.0-alpha.1",
     "tap-mocha-reporter": "^3.0.6",
-    "tree-kill": "^1.2.0",
-    "ts-node": "^4.1.0",
-    "webdriver-manager": "^12.0.6",
-    "yargs": "^11.0.0"
+    "tap-spec": "^4.1.1",
+    "ts-node": "^4.1.0"
   },
   "scripts": {
     "clean": "node ../node_modules/rimraf/bin.js ./wwwroot/dist ./obj/js",
     "build": "npm run clean && npm run build:lint && npm run build:tsc && npm run build:rollup",
     "build:lint": "node ../node_modules/tslint/bin/tslint -c ../tslint.json -p ./tsconfig.json",
     "build:tsc": "node ../node_modules/typescript/bin/tsc --project ./tsconfig.json",
     "build:rollup": "node ../node_modules/rollup/bin/rollup -c",
-    "rawtest": "ts-node --project ./run-tests.tsconfig.json ./run-tests.ts --browser chrome -h",
-    "test": "npm run rawtest | faucet"
+    "pretest": "npm run build",
+    "rawtest": "ts-node --project ./selenium/tsconfig.json ./selenium/run-tests.ts",
+    "test": "ts-node --project ./selenium/tsconfig.json ./selenium/run-tests.ts | tap-spec",
+    "ci-test-Debug": "npm run pretest && ts-node --project ./selenium/tsconfig.json ./selenium/run-tests.ts | tap-spec",
+    "ci-test-Release": "npm run pretest && ts-node --project ./selenium/tsconfig.json ./selenium/run-tests.ts --release | tap-spec"
```

Comment:
This whole `ci-test-[Configuration]` is unpleasant and hacky. I'll have a better solution before merging. The problem is that because we pipe to `tap-spec`, we can't just use `npm run ci-test -- --configuration Release` because NPM just tacks the extra args on to the end of the string meaning they don't go to our test script. I'm thinking maybe environment variables is a solution.